src/core/Breadcrumb/Breadcrumb/Breadcrumb.md

```diff
@@ -1,5 +1,9 @@
 The `<Breadcrumb>` component is used to let the user know their current location in a web service. It is rendered as an HTML `nav` element.
 
+### Accessibility Notes
```

Comment:
```suggestion
### Accessibility notes
```

Reply:
Because of these headings the "examples" section looks like it belongs to the accessibility notes, which is quite misleading.